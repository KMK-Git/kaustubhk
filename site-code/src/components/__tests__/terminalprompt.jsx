import React from 'react';
import renderer from 'react-test-renderer';
import TerminalPrompt from '../terminalprompt';

const { act } = renderer;
const user = 'kaustubh@kaustubhk.com:';
const location = '~';

describe('TerminalPage', () => {
  it('renders correctly', async () => {
    let tree;
    act(() => {
      tree = renderer
        .create(<TerminalPrompt user={user} location={location} text="cat aboutme.txt" />);
    });
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
