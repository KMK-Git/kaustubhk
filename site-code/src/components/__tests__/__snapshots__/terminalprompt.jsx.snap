// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TerminalPage renders correctly 1`] = `
<span
  style={
    Object {
      "display": "inline-block",
      "marginTop": "2px",
    }
  }
>
  <div
    style={
      Object {
        "color": "#87d441",
        "display": "inline-block",
        "whiteSpace": "normal",
        "wordBreak": "break-all",
        "wordWrap": "break-word",
      }
    }
  >
    kaustubh@kaustubhk.com:
  </div>
  <div
    style={
      Object {
        "color": "#6d85a9",
        "display": "inline-block",
        "whiteSpace": "normal",
        "wordBreak": "break-all",
        "wordWrap": "break-word",
      }
    }
  >
    ~
  </div>
  <div
    style={
      Object {
        "color": "#dddddd",
        "display": "inline-block",
        "whiteSpace": "normal",
        "wordBreak": "break-all",
        "wordWrap": "break-word",
      }
    }
  >
    $
  </div>
  Â 
  <div
    style={
      Object {
        "color": "#dddddd",
        "display": "inline-block",
        "whiteSpace": "normal",
        "wordBreak": "break-all",
        "wordWrap": "break-word",
      }
    }
  >
    cat aboutme.txt
  </div>
</span>
`;
